pyproject.toml

```diff
@@ -1,7 +1,56 @@
+[tool.poetry]
+name = "langchain"
+version = "0.0.26"
+description = "Building applications with LLMs through composability"
+authors = []
+license = "MIT"
+readme = "README.md"
+repository = "https://www.github.com/hwchase17/langchain"
+
+[tool.poetry.dependencies]
+python = ">=3.8.1,<4.0"
```

Comment:
> (small) possible to also support 3.7 until its EOL? im guessing maybe the reason you didn't is manifest-ml doesnt?

`numpy` and `flake8` also have dropped support for 3.7, the former end of last year and the latter just recently.

Happy to loosen these constraints to find a compatible, older version.

For `manifest-ml`, I think we can include it as an optional dependency for a specific version of Python, but not 100% sure yet.

Reply:
oh wow i didnt know numpy did. okay im fine moving on